admin/base/settings/defaults.py

```diff
@@ -20,6 +20,15 @@
 DEBUG = osf_settings.DEBUG_MODE
 DEBUG_PROPAGATE_EXCEPTIONS = True
 
+# secure mode for using https in local development
+SECURE = osf_settings.SECURE_MODE
+LOCAL = osf_settings.LOCAL_MODE
+
+# session:
+SESSION_COOKIE_HTTPONLY = True
```

Comment:
Can you please add a TODO to change this to `settings.SESSION_COOKIE_HTTPONLY` in the future? I understand the reasons for hard-coding the value for the admin tool... but we don't want to forget to make things consistent later.

Be sure to cite the relevant JIRA ticket number in the source code comment, so we can link the TODO to a known issue in our tracker.